tox.ini

```
@@ -33,6 +34,9 @@ commands =
     behave-lxd-16.04: behave -v {posargs} --tags="series.xenial,series.all" --tags="~upgrade"
     behave-lxd-18.04: behave -v {posargs} --tags="series.bionic,series.all" --tags="~upgrade"
     behave-lxd-20.04: behave -v {posargs} --tags="series.focal,series.all" --tags="~upgrade"
+    behave-vm-16.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.vm" --tags="series.xenial"
+    behave-vm-18.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.vm" --tags="series.bionic"
```

These targets should match series.all as well so we don't skip features/attach_validtoken.feature line 76 which decorates with a series.all instead of series.bionic specifically.
```suggestion
    behave-vm-18.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.vm" --tags="series.all,series.bionic"
```